config/backend-config/multitenant-workspace-config.go

```diff
@@ -63,13 +63,10 @@ func (workspaceConfig *MultiTenantWorkspaceConfig) Get(workspaces string) (Confi
 
 // getFromApi gets the workspace config from api
 func (workspaceConfig *MultiTenantWorkspaceConfig) getFromAPI(workspaceArr string) (ConfigT, bool) {
-	url := fmt.Sprintf("%s/hostedWorkspaceConfig?fetchAll=true", configBackendURL)
-	//TODO : Uncomment it and add tests once we have cluster managers ready
-	//To support existing multitenant behaviour
-	// url := fmt.Sprintf("%s/multitenantWorkspaceConfig?ids=[%s]", configBackendURL, workspaceArr)
-	// workspacesString := ""
-	// url = url + workspacesString
-	// url = url + "&fetchAll=true"
+	url := fmt.Sprintf("%s/multitenantWorkspaceConfig?ids=[%s]", configBackendURL, workspaceArr)
```

Comment:
I agree with this suggestion.

Regarding `multi-valued query parameters`:
1. I am not sure if the control plane will accept something like this.
2. Given the number of workspaceIDs we expect, it might have a big overhead. 

cc @cisse21

Reply:
The current way is the contract we have with control-plane ..... would require changes from CP side if we need to incorporate such changes. In production we were expecting not to have more than 200-300 workspaces on a single rudder-server so might not be a big overhead I guess

Reply:
https://rudderlabs.slack.com/archives/C02A3UPEWMU/p1644392187564439

Reply:
Only implemented the first suggestion for now.

Reply:
I have the following issue:
```
  Got: https://api.rudderlabs.com/multitenantWorkspaceConfig?fetchAll=true&ids=%5BtestToken%5D (string)
  Want: is equal to https://api.rudderlabs.com/multitenantWorkspaceConfig?ids=[testToken]&fetchAll=true (string)
  ```